### PR TITLE
Changed to distribute via pypi and updated docs

### PR DESCRIPTION
**Pull request by stevenca**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+	echo -e "\nRun 'make format' to automatically make changes.\n" ;\
+	exit 1 ;\
+	fi ;\
+	)
+
+format: $(VENV_BIN)/activate ## Format code
+	$(VENV_BIN)/yapf --style=yapf.ini -i -r *.py $(PYDIRS)
+
+pylint: $(VENV_BIN)/activate ## Run pylint
+	$(VENV_BIN)/pylint --output-format=parseable --rcfile .pylintrc *.py $(PYDIRS)
```
Comment (jlothian) [inline Makefile:54]: change to:
$(VENV_BIN)/pylint --output-format=parseable --rcfile .pylintrc *.py $(SRCFILES)

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,101 @@
+# Makefile
+include envfile
+PYTHON_EXE = python3
+TOPDIR = $(shell git rev-parse --show-toplevel)
+PYDIRS="ansible"
+VENV = venv_ansible_virl
+VENV_BIN=$(VENV)/bin
+SRC_FILES := $(shell find ansible -name \*.py)
+SPHINX_DEPS := $(shell find docs/src)
```
[the remark this answers is not in the document]
Comment (stevenca) [reply Makefile]: Removed lines 6 & 7, but make clean/dev  broke when I removed 8 & 9.

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+NON_PYTHON_LIBS := $(shell ls | grep -v vmanage)
```
[the remark this answers is not in the document]
Comment (stevenca) [reply Makefile]: Removed 8-10, left 7.

File: tox.ini [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+envlist = py3
+
+[testenv:py3]
+passenv = DYLD_LIBRARY_PATH
+deps = -r{toxinidir}/requirements.txt
+       -r{toxinidir}/test-requirements.txt
+commands =
+    py.test -v --cov vmanage --cov-report term-missing tests {posargs}
+    pylint --output-format=parseable --rcfile .pylintrc tests/ vmanage/
+    flake8 --max-line-length=120 tests/ vmanage/ --exclude=.ropeproject
```
[the remark this answers is not in the document]
Comment (stevenca) [reply tox.ini]: Done.

**Pull request merged**